add Admin questions (#193)
FidelusAleksander/ghcertified: content/questions/admin/question-094.md

---

> question: "How do you use GitHub Packages in workflows, such as with GitHub Actions or other CI/CD tools?"

Reviewer: ```suggestion
question: "What is required to download or publish a GitHub Package within workflows, such as with GitHub Actions or other CI/CD tools?"
```
